pkgs/by-name/qu/quesoglc/package.nix

```
@@ -26,8 +28,11 @@ stdenv.mkDerivation rec {
     fribidi
     libX11
   ];
+
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-pointer-types";
```

```suggestion
  # required for cross builds
  configureFlags = [
    "ac_cv_func_malloc_0_nonnull=yes"
    "ac_cv_func_realloc_0_nonnull=yes"
    "ac_cv_func_memcmp_working=yes"
  ];

  # unbreak build on GCC 14
  env.NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-pointer-types";
```
Could fix cross builds while we are here.